src/KernelAbstractions.jl

```diff
@@ -535,6 +535,14 @@ function priority!(::Backend, prio::Symbol)
     return nothing
 end
 
+
+"""
+    ongpu()
+
+Returns if this kernel is currently executing on a GPU.
+"""
+function isongpu end
```

Comment:
there's a mismatch here between the docstring and the function name. The method in `src/cpu` also calls in `ongpu`